webapp/src/components/training/TrainingInformation.vue

```diff
@@ -51,8 +51,9 @@
       </IconCardSmall>
     </div>
 
-    <!-- Training and validation loss charts -->
-    <div
+    <DropdownCard :initiallyOpen="currentOpen" @click="toggleAdvancedInfo">
```

Comment:
consider renaming `currentOpen` to `initiallyOpen`, then you can use the short version of the attribute, which is a bit nicer

```suggestion
    <DropdownCard :initiallyOpen @click="toggleAdvancedInfo">
```